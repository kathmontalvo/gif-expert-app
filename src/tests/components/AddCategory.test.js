import React from 'react';
const { shallow } = require("enzyme")
const { AddCategory } = require("../../components/AddCategory")

describe('Pruebas en AddCategory', () => {

    const setCategories = () => {
        
    }
    test('debe mostrarse correctaente', () => {
        const wrapper = shallow(<AddCategory setCategories={setCategories} />);
        expect(wrapper).toMatchSnapshot();
    })
    
})
