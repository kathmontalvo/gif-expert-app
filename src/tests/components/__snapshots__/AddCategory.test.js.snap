// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en AddCategory debe mostrarse correctaente 1`] = `
<form
  onSubmit={[Function]}
>
  <input
    onChange={[Function]}
    placeholder="Escribe aqui..."
    type="text"
    value=""
  />
</form>
`;
